models/dashboard_subsidio_sppo/sumario_servico_tipo_viagem_dia.sql

```diff
@@ -7,7 +7,7 @@ WITH
     consorcio,
     servico
   FROM
-    `rj-smtr`.`projeto_subsidio_sppo`.`viagem_planejada`
+    {{ ref("viagem_planejada") }}
   WHERE
     `data` <= DATE( "{{ var("end_date") }}" )
     AND distancia_total_planejada > 0 ),
```

Comment:
```suggestion
    AND distancia_total_planejada is not null ),
```

Correção: quando um serviço não tem KM total planejada (i.e. não é subsdiado), vamos deixar o valor nulo

Reply:
Ok, vou deixar uma condição que atenda a ambos casos